tests/test.py

```diff
@@ -16,6 +21,48 @@
 class TestDSS(unittest.TestCase):
     def setUp(self):
         self.app = dss.create_app().app.test_client()
+        self.sre = re.compile("^assert(.+)Response")
+
+    def assertResponse(
+            self,
+            method: str,
+            path: str,
+            expected_code: int,
+            expected_headers: typing.Dict={},
+            json_response_tester: typing.Optional[typing.Callable[[typing.Any], bool]]=None,
```

Comment:
I would suggest using this pattern instead:
```
def assert_response(..., expect_json=True, **kwargs):
    if "body" in kwargs and isinstance(kwargs["body"], dict):
        headers["content-type"] = "application/json"
        kwargs["body"] = json.dumps(kwargs["body"])
    if expect_json:
        return res.json() # or whatever means we must use to unpack json for now
    else:
        return res
```
This avoids boilerplate and allows me to chain assertions directly or tersely on the return value.

Alternatively, we could do another wrapper around this helper to implement this (e.g. assert_json_response).